### DISCUSSION TITLE
Vulkan or OpenGL?

### DISCUSSION THREAD

**Discussion by neoOpus** [vosen/ZLUDA#220] [Ideas]
Have you considered seeking support from Khronos Group, OpenGL, or smartphone GPU manufacturers to take this technology to the next level and benefit more people?

**Comment by vosen** [comment]
Khronos has their own GPU API/language called SYCL. I don't think they would be interested in ZLUDA.
On smartphones there is not so much existing CUDA code, so probably nt much interest either
